Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocListImplementation/Rendering/BocCellAttributeRenderingContextTest.cs

```diff
@@ -0,0 +1,229 @@
+﻿// // This file is part of the re-motion Core Framework (www.re-motion.org)
+// // Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// //
+// // The re-motion Core Framework is free software; you can redistribute it
+// // and/or modify it under the terms of the GNU Lesser General Public License
+// // as published by the Free Software Foundation; either version 2.1 of the
+// // License, or (at your option) any later version.
+// //
+// // re-motion is distributed in the hope that it will be useful,
+// // but WITHOUT ANY WARRANTY; without even the implied warranty of
+// // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// // GNU Lesser General Public License for more details.
+// //
+// // You should have received a copy of the GNU Lesser General Public License
+// // along with re-motion; if not, see http://www.gnu.org/licenses.
+// //
+using System.Web;
+using System.Web.UI;
+using Moq;
+using NUnit.Framework;
+using Remotion.ObjectBinding.Web.Services;
+using Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation;
+using Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation.Rendering;
+using Remotion.Web;
+
+namespace Remotion.ObjectBinding.Web.UnitTests.UI.Controls.BocListImplementation.Rendering
+{
+  [TestFixture]
+  public class BocCellAttributeRenderingContextTest
+  {
+    private HtmlHelper _html;
+    private StubColumnDefinition _columnDefinition;
+    private Mock<HttpContextBase> _httpContextStub;
+    private Mock<IBocList> _bocListStub;
+    private BocColumnRenderingContext<StubColumnDefinition> _renderingContext;
+    private BocCellAttributeRenderingContext<StubColumnDefinition> _attributeRenderingContext;
+
+    [SetUp]
+    public void SetUp ()
+    {
+      _html = new HtmlHelper();
+      _columnDefinition = new StubColumnDefinition();
+      _httpContextStub = new Mock<HttpContextBase>();
+      _bocListStub = new Mock<IBocList>();
+      _renderingContext = new BocColumnRenderingContext<StubColumnDefinition>(
+          new BocColumnRenderingContext(
+              _httpContextStub.Object,
+              _html.Writer,
+              _bocListStub.Object,
+              BusinessObjectWebServiceContext.Create(null, null, null),
+              _columnDefinition,
+              17,
+              23));
+      _attributeRenderingContext = new BocCellAttributeRenderingContext<StubColumnDefinition>(_renderingContext);
+    }
+
+    [Test]
+    public void AddAttributeToRender_WithStringNameAndNullableStringValueAndDefaultEncodeTrue_EncodesAttributeValue ()
```

Comment:
in the text, the "nullablestring" instead of "nullable" isn't necessary. It does not help to distinguish the test cases. We don't need to revert those, it's for future reference :)